platform/security/core/security-core-services/src/main/java/ddf/security/samlp/impl/LogoutMessageImpl.java

```diff
@@ -135,8 +134,7 @@ public LogoutRequest extractSamlLogoutRequest(String samlLogoutResponse)
   @Override
   public LogoutRequest buildLogoutRequest(
       String nameIdString, String issuerOrEntityId, List<String> sessionIndexes) {
-    return buildLogoutRequest(
-        nameIdString, issuerOrEntityId, getUuidGenerator().generateUuid(), sessionIndexes);
+    return buildLogoutRequest(nameIdString, issuerOrEntityId, generateId(), sessionIndexes);
```

Comment:
For other reviewers: it took me a minute to see what had actually changed; this is the line.